scripts/pylib/twister/twisterlib/runner.py

```diff
@@ -1041,20 +1246,55 @@ def _sanitize_zephyr_base_from_files(self):
             with open(file_path, "wt") as file:
                 file.write(data)
 
+    @staticmethod
+    def _add_instance_testcases_to_status_counts(instance, results, decrement=False):
+        increment_value = -1 if decrement else 1
+        for tc in instance.testcases:
+            match tc.status:
```

Comment:
btw. without a python version guard this will raise errors on python < 3.10

Reply:
> In Zephyr Getting Started Guide the current minimum required version is python 3.10.

@arikgreen  more context https://github.com/zephyrproject-rtos/zephyr/pull/78671#issuecomment-2363529256